rero_ils/modules/acq_order_lines/dumpers.py

```diff
@@ -66,3 +67,29 @@ def dump(self, record, data):
         }
         data['document'] = {k: v for k, v in data['document'].items() if v}
         return data
+
+
+class AcqOrderLineNotificationDumper(InvenioRecordsDumper):
+    """Order line dumper class for acquisition order."""
+
+    def dump(self, record, data):
+        """Dump an AcqOrderLine instance for acquisition order notification.
```

Comment:
```suggestion
        """Dump an acquisition order line instance for acquisition order notification.
```

I do prefer natural language in documentation sentences. :slightly_smiling_face: Of course, my opinion stays the same for other occurrences.

Reply:
It' debate time ! 
My feeling is that I would mention this dumper class should be use with an object 'AcqOrderLine' class instance. If I choose your proposal, when I will read this comment in a couple of year, I could think I could use this dumper with any class representing something like a "line for a acquisition order". Is this a comment, Is this an address block, ...